packages/backend/src/app.ts

```diff
@@ -393,7 +393,11 @@ export class App {
     )
 
     let tenantApiSignatureResult: TenantApiSignatureResult
-    if (this.config.env !== 'test') {
+    if (this.config.env === 'test') {
+      const tenantService = await this.container.use('tenantService')
+      const tenant = await tenantService.get(this.config.operatorTenantId)
+      tenantApiSignatureResult = { tenant, isOperator: true }
```

Comment:
ah, we basically got to the same conclusion :)

I ended up setting a `tenant-id` header in the apollo client during tests: https://github.com/interledger/rafiki/pull/3206/files#diff-5b481c1a1464d3e282df7102a0cb5882701fc2b2af81037e203812e13acf0176R414-R433

Both work, I'm trying to think whether we would actually want to be able to pass in a different tenant id during tests...

Reply:
Might there be tests to verify that requests are properly permissioned between an operator making a request versus a tenant?

Reply:
Good point @njlie , I will add those in. I added a new error called: `WalletAddressError.InvalidTenantIdNotAllowed`:

```node
    const tenantId = tenantIdToProceed(
      ctx.isOperator,
      ctx.tenant.id,
      args.input.tenantId
    )
    if (!tenantId) {
      const err = WalletAddressError.InvalidTenantIdNotAllowed
      throw new GraphQLError(errorToMessage[err], {
        extensions: {
          code: errorToCode[err]
        }
      })
    }
```